processcheck/topProcess.py

```diff
@@ -1,10 +0,0 @@
-def topProcess(numOfProcesses):
-    try:
```

Comment:
This shouldn't be necessary although I understand it was an attempt to debug what was going on.